includes/settings/controls.php

```diff
@@ -157,12 +156,12 @@ private static function select( array $field ) {
 			<?php endif; ?>
 
 			<?php foreach ( $field['options'] as $value => $label ) : ?>
-				<option value="<?php echo esc_attr( $value ); ?>"<?php selected( $value, $old_value ); ?>><?php echo $label; ?></option>
+				<option value="<?php echo esc_attr( $value ); ?>"<?php selected( $value, $old_value ); ?>><?php echo esc_html( $label ); ?></option>
```

Comment:
I think that the selected() also prints a value no?

Reply:
Yes you are right will escape the selected function